OgcApi.Net/Controllers/CollectionsController.cs

```diff
@@ -280,14 +284,20 @@ public ActionResult<Collection> Get(string collectionId)
 
             try
             {
+                var queryCollection = HttpUtility.ParseQueryString(Request.QueryString.Value);
+                var propertyFilter = queryCollection.AllKeys
+                    .Where(key => properties.Any(property => property == key))
+                    .ToDictionary(key => key, key => queryCollection[key]);
```

Comment:
```suggestion
                Dictionary<string, string> propertyFilter = null;
		if (properties is {Count: > 0})
		{
			propertyFilter = Request.Query.Keys.Where(properties.Contains)
				.ToDictionary(key => key, key => Request.Query[key].First());
		}
```

It is necessary to test a situation in which several identical parameters come to the API input. For example, a=1&a=2